daemons/health-check-daemon/app.py

```diff
@@ -0,0 +1,8 @@
+from upload.common.logging import get_logger
+from upload.lambdas.health_check.health_check import HealthCheck
+
+logger = get_logger(__name__)
+
+
+def health_check(event, context):
+    HealthCheck().health_check()
```

Comment:
(nit). This makes it sound like you are running a health check on a health check. Can we rename the function to something like run_upload_service_health_check?

Reply:
👍